api/auth/auth.controller.js

```diff
@@ -44,7 +44,10 @@ class AuthController {
     return bcrypt.compareSync(plaintext, hash)
   }
 
-  signin(email, password) {
+  signin(request) {
+    const { payload } = request || {}
```

Comment:
Destructuring and default value setting can be made at the function's parameter place.